<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>&amp;Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>&amp;Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveAsButton">
           <property name="text">
            <string>Save &amp;as</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>Play /&amp; Pause</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stillButton">
           <property name="text">
            <string>Play/still</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QFrame" name="frame1">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="CamName1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Cam1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="fs1">
             <property name="text">
              <string>Fullscreen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="VideoWidget" name="cam1" native="true"/>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="ts1">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame0">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLineEdit" name="CamName0">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Cam0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="fs0">
             <property name="text">
              <string>Fullscreen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="VideoWidget" name="cam0" native="true"/>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="ts0">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMedia_File_Load">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSelect_Media_File"/>
    <addaction name="actionSelect_Fibex_File"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuSelect_Media">
    <property name="title">
     <string>Select &amp;Widgets</string>
    </property>
    <addaction name="actionCAM0"/>
    <addaction name="actionCAM1"/>
    <addaction name="actionCAM2"/>
    <addaction name="actionCAM3"/>
    <addaction name="actionFlexRay0"/>
    <addaction name="actionFlexRay1"/>
    <addaction name="actionCAN0"/>
    <addaction name="actionCAN1"/>
    <addaction name="actionAE0"/>
    <addaction name="actionAE1"/>
   </widget>
   <widget class="QMenu" name="menuPatterns">
    <property name="title">
     <string>&amp;Patterns</string>
    </property>
    <addaction name="actionReal"/>
    <addaction name="actionPattern_1"/>
    <addaction name="actionPattern_2"/>
    <addaction name="actionPattern_3"/>
    <addaction name="actionPattern_4"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionEdit_config"/>
    <addaction name="actionLoad_config_file"/>
    <addaction name="actionSave_config_file"/>
   </widget>
   <addaction name="menuMedia_File_Load"/>
   <addaction name="menuSelect_Media"/>
   <addaction name="menuPatterns"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSelect_Media_File">
   <property name="text">
    <string>Select &amp;Media File</string>
   </property>
  </action>
  <action name="actionSelect_Fibex_File">
   <property name="text">
    <string>Select &amp;Fibex File</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionReal">
   <property name="text">
    <string>Real</string>
   </property>
  </action>
  <action name="actionPattern_1">
   <property name="text">
    <string>Pattern 1</string>
   </property>
  </action>
  <action name="actionPattern_2">
   <property name="text">
    <string>Pattern 2</string>
   </property>
  </action>
  <action name="actionPattern_3">
   <property name="text">
    <string>Pattern 3</string>
   </property>
  </action>
  <action name="actionPattern_4">
   <property name="text">
    <string>Pattern 4</string>
   </property>
  </action>
  <action name="actionCAM0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CAM0</string>
   </property>
  </action>
  <action name="actionCAM1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CAM1</string>
   </property>
  </action>
  <action name="actionFlexRay0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FlexRay0</string>
   </property>
  </action>
  <action name="actionFlexRay1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FlexRay1</string>
   </property>
  </action>
  <action name="actionCAN0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CAN0</string>
   </property>
  </action>
  <action name="actionCAN1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CAN1</string>
   </property>
  </action>
  <action name="actionAE0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>AE0</string>
   </property>
  </action>
  <action name="actionAE1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>AE1</string>
   </property>
  </action>
  <action name="actionCAM2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CAM2</string>
   </property>
  </action>
  <action name="actionCAM3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CAM3</string>
   </property>
  </action>
  <action name="actionEdit_config">
   <property name="text">
    <string>Edit config</string>
   </property>
  </action>
  <action name="actionLoad_config_file">
   <property name="text">
    <string>Load config file</string>
   </property>
  </action>
  <action name="actionSave_config_file">
   <property name="text">
    <string>Save config file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>VideoWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
